File: Assignment24/C/Program1.c
// PS - Accept string from user and accept 1 character. check whether that character is present in string or not.

#include <stdio.h>
#include<stdbool.h>

bool CheckChar(char *str, char ch)
{
    bool bFlag = false;
    while (*str != '\0')
    {
        if (*str == ch)
        {
            bFlag = true;
            break;
        }
        str++;
    }
    return bFlag;
}

int main()
{
    char Arr[20];
    char ch = '\0';
    bool bRet = false;

    printf("Enter string \n");
    scanf("%[^'\n']s", Arr);

    printf("Enter character \n");
    scanf(" %c", &ch);

    bRet = CheckChar(Arr, ch);

    if(bRet == true) {
        printf("Character is present \n");
    } else {
        printf("Character is not present \n");
    }

    return 0;
}

File: Assignment36/Java/Program1.java
// PS - write java program which accept N numbers from user and accept one more number and check whether that number is present or not.
import java.util.*;

class ArrayX
{
    public boolean CheckNum(int Arr[], int No)
    {
        boolean bFlag = false;
        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] == No)
            {
                bFlag = true;
            }
        }
        return bFlag;
    }
}

class Program1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements");
        int iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter elements of array");
        for(int iCnt = 0; iCnt < iSize; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }

        System.out.println("Enter number that you want to search");
        int No = sobj.nextInt();

        ArrayX aobj = new ArrayX();
        boolean bret = aobj.CheckNum(Arr, No);

        if(bret == true)
        {
            System.out.println("Number is present");
        }
        else
        {
            System.out.println("Number is not present");
        }
    }
}